### PR TITLE
Add mechanism to prepare QA builds

### PR DESCRIPTION
**Pull request by pazaan**
Also add `DEBUG_ERROR=true` to `dev` builds in order to make Blob downloads available.

### CODE REVIEW

**Comment by pazaan** [comment]
@krystophv - I considered also adding an automatic commit at the end, to make the QA build completely consistent (ie, one can't change another file, then commit and push). Thoughts?

File: package.json [hunk 1]
```diff
@@ -199,6 +203,7 @@
     "fbjs-scripts": "0.7.1",
     "file-loader": "0.11.1",
     "flux-standard-action": "1.2.0",
+    "git-describe": "^4.0.2",
```
Comment (krystophv) [inline package.json]: These just need to be exact versions.

File: scripts/prepare-qa-build.js [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+const gitInfo = gitDescribeSync();
+if (gitInfo.dirty) {
+  console.log('Please start with a clean repo');
+  shell.exit(1);
+}
+
+const qaVersion = gitInfo.semverString.replace('+', '.');
+
+(async() => {
```
Comment (krystophv) [inline scripts/prepare-qa-build.js]: This causes an error on `node 7.4.0` without running this script command through `babel` first (accomplished in the other npm scripts by using `babel-register`).

Comment (pazaan) [inline scripts/prepare-qa-build.js]: Updated `package.json` to fix. Thanks!

**Comment by krystophv** [comment]
LGTM 👍

**Pull request merged**